staff/groups/aleti/task-app-3.0.0/task-app/src/logic.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
this file is not used, in fact has no export. DELETE unnecessary files (clean code).

Reply:
done!